### DISCUSSION TITLE
Installing coin-or-Cbc on Alpine distribution

### DISCUSSION THREAD

**Discussion by kennylajara** [coin-or/Cbc#437] [Q&A]
I looked at https://projects.coin-or.org/Cbc and I found instructions on how to install it on several OS and Linux distributions, but I couldn't find how to install in on Alpine.

I tried the git and svn installations but I couldn't success.

Can you help me on that?

**Comment by tkralphs** [comment]
https://projects.coin-or.org/Cbc is the old wiki and the instructions there are outdated (as clearly stated there). This repository is where Cbc is now maintained and where you should come for up-to-date instructions. As far as I know, the instructions for building from source linked from the README should work on any Linux distro, so we'll need a lot more details about what you did and why it failed if you want help.

**Reply by tkralphs** [reply]
I converted this from an Issue to a Discussion.

**Reply by ldeluigi** [reply]
Can you distribute Cbc for apk?
